app/models/music.rb

```diff
@@ -0,0 +1,17 @@
+require 'rspotify'
+
+class Music < ActiveRecord::Base
+  def self.search(type, id)
+    if type.downcase == "artist"
```

Comment:
I think this would be a great place to use a case/when statement instead of if/elsif to set the type of search. Both ways work.